src/Grid/Grid.js

```diff
@@ -107,7 +107,7 @@ function Grid(element, options) {
 
   // Throw an error if the container element is not body element or does not
   // exist within the body element.
-  if (!document.body.contains(element)) {
+  if (!element instanceof HTMLElement || element === document.documentElement) {
```

Comment:
The original intention was to make sure that the element is in the DOM tree, which this new check does not do anymore.

```javascript
// The problem
document.createElement('div') instanceof HTMLElement // true
```

I'm thinking something like this would suit better:

```javascript
var isElementInDom = element.getRootNode ? element.getRootNode({ composed: true }) === document : document.body.contains(element);
if (!isElementInDom) {
  throw new Error('Container element must be an existing DOM element')
}
```

Reply:
I've been thinking about your idea to simply remove the checking attached or not the element to the dom. And decided to check only the type of the provided object. But if we still need to check attach your version will suit better.